x-pack/plugins/observability_solution/observability/server/lib/rules/custom_threshold/custom_threshold_executor.test.ts

```diff
@@ -2223,6 +2223,1692 @@ describe('The custom threshold alert type', () => {
   });
 });
 
+describe('The custom threshold rule type with last_value aggregation', () => {
+  setup();
+
+  describe('querying the entire infrastructure', () => {
+    beforeEach(() => jest.clearAllMocks());
+    afterAll(() => clearInstances());
+    const instanceID = '*';
+    const execute = (comparator: COMPARATORS, threshold: number[], sourceId: string = 'default') =>
+      executor({
+        ...mockOptions,
+        services,
+        params: {
+          ...mockOptions.params,
+          sourceId,
+          criteria: [
+            {
+              ...customThresholdLastValueCriterion,
+              comparator,
+              threshold,
+            },
+          ],
+        },
+      });
+    const setResults = (
+      comparator: COMPARATORS,
+      threshold: number[],
+      shouldFire: boolean = false,
+      isNoData: boolean = false
+    ) =>
+      setEvaluationResults([
+        {
+          '*': {
+            ...customThresholdLastValueCriterion,
+            comparator,
+            threshold,
+            currentValue: 1.0,
+            timestamp: new Date().toISOString(),
+            shouldFire,
+            isNoData,
+            bucketKey: { groupBy0: '*' },
+          },
+        },
+      ]);
+    test('alerts as expected with the > comparator', async () => {
```

Comment:
Q: Since we are mocking the evaluation result, do you see a benefit in checking each comparator for different aggregations?

Reply:
maybe no benefit, I took reference of existing tests with `average` aggregation where we are checking each comparator. I will keep these tests to have consistency in both describe blocks.